lib/build-tokens.js

```diff
@@ -9,13 +10,17 @@ const { createIndexCssFile } = require('../tokens/utils');
  * @param {string[]} commandArgs - Command line arguments for building tokens.
  * @param {string} [commandArgs.build-dir='./build/'] - The directory where the build output will be placed.
  * @param {string} [commandArgs.source] - The source directory containing JSON token files.
+ * @param {string} [commandArgs.base-theme] - The base theme to use from Paragon if named differently than the theme.
  * @param {boolean} [commandArgs.source-tokens-only=false] - Indicates whether to include only source tokens.
  * @param {string|string[]} [commandArgs.themes=['light']] - The themes (variants) for which to build tokens.
+ * @param {boolean} [commandArgs.all-themes] - Indicated whether to process all themes.
  */
 async function buildTokensCommand(commandArgs) {
   const defaultParams = {
-    themes: ['light'],
+    themes: 'light',
```

Comment:
Any specific reason to switch from `themes` argument as a list, to a string and splitting it later in `:48` ?

If so, please also update the `@param` doc in `:15` to reflect the same

Reply:
The reason is that currently specifying a single theme via `--theme` is simply broken. You will get an error if you run the command. 

The reason for that is that if you specify a single theme as `--theme light` the value of themes above will be the string "light" and so the `themes.forEach` later on will fail. You'll need to either go with the default value, or specify themes twice as:

`paragon build-tokens --themes light --themes dark`

This change makes it possible to specify multiple themes as:

`paragon build-tokens --themes "light  dark"`

This is also a bit clumsy, but is more consistent. I'll update the docstring and also make this more robust to support the old syntax.